lib/qam.pm

```diff
@@ -75,7 +75,8 @@ sub is_patch_needed {
 
 sub add_repo_if_not_present {
     my ($url, $name) = @_;
-    my $gpg = get_var('BUILD') =~ m/^MR:/ ? "-G" : "";
+    my $gpg = "";
+    $gpg = "-G" if (get_var('BUILD') =~ m/^MR:/ || get_var('MU_REPOS_NO_GPG_CHECK'));
```

Comment:
```suggestion
    $gpg = "-G" if (get_var('MU_REPOS_NO_GPG_CHECK'));
```
I'd prefer not to match `BUILD` - the `MU_REPOS_NO_GPG_CHECK` seems more intuitive for me.

Or maybe you can do:
```suggestion
    $gpg = "-G" if (get_var('BUILD') =~ m/^MR:/ && get_var('MU_REPOS_NO_GPG_CHECK'));
```

Anyways, please don't forget about `variables.md` :smile:

Reply:
I would prefer not to remove the logic that was before, I don't want to break things. Also, I added it to variables.md already :)